tests/watch_test.go

```diff
@@ -0,0 +1,65 @@
+package test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"go.uber.org/zap/zapcore"
+
+	xlineapi "github.com/xline-kv/go-xline/api/xline"
+	"github.com/xline-kv/go-xline/client"
+	"github.com/xline-kv/go-xline/xlog"
+)
+
+func TestWatch(t *testing.T) {
+	xlog.SetLevel(zapcore.WarnLevel)
+
+	curpMembers := []string{"172.20.0.3:2379", "172.20.0.4:2379", "172.20.0.5:2379"}
+
+	client, err := client.Connect(curpMembers)
+	assert.NoError(t, err)
+	kvClient := client.Kv
+	watchClient := client.Watch
+
+	t.Run("watch_should_receive_consistent_events", func(t *testing.T) {
+		watcher, err := watchClient.Watch()
+		assert.NoError(t, err)
+
+		err = watcher.Send(&xlineapi.WatchRequest{
+			RequestUnion: &xlineapi.WatchRequest_CreateRequest{
+				CreateRequest: &xlineapi.WatchCreateRequest{
+					Key: []byte("watch01"),
+				},
+			},
+		})
```

Comment:
We can add a method to simplify the sending of this request.

Reply:
This issue is related to client ease of use, and is planned to be optimized in a later pr.